config/config.go

```diff
@@ -576,6 +586,10 @@ func NewConfig() (c *Config) {
 			VersionLabelName:   "version",
 		},
 		KialiFeatureFlags: KialiFeatureFlags{
+			CertificatesInformationIndicators: CertificatesInformationIndicators{
```

Comment:
ask @leandroberetta  if this is OK. I wonder if he omited the defaults here for a reason?

Reply:
shoot, nm.. this is just re-ordering it.